src/game_scene.cpp

```diff
@@ -21,7 +21,15 @@ GameScene::GameScene(ConsoleUI::UIManager &uiManager, std::function<void()> goBa
 
 void GameScene::update()
 {
-    // TO DO
+
+}
+
+void GameScene::init()
+{
+    Player user = Player(100, 100, 5, generateDeck(20));
+    Player bot = Player(100, 100, 5, generateDeck(20));
+
+    GameScene::game = setUp(user, bot);
```

Comment:
Should this be `GameScene::game = Game(user, bot);`? or has setUp been created somewhere else but not committed? Currently doesn't compile due to `setUp` not being defined

Reply:
I removed `setUp()` because it was redundant. It should be `Game(user, bot)`.